foundationdb-kubernetes-sidecar/sidecar.py

```diff
@@ -315,9 +354,15 @@ def start(cls):
         This method starts the server.
         """
         config = Config.shared()
-        (address, port) = config.bind_address.split(":")
+        port_index = config.bind_address.rindex(":")
+        address = config.bind_address[:port_index]
+        port = config.bind_address[port_index+1:]
         log.info(f"Listening on {address}:{port}")
-        server = ThreadingHTTPServer((address, int(port)), cls)
+
+        if ":" in address:
```

Comment:
Please use `ipaddress.ip_address(address).version == 6` instead of checking for `:` the latter is error prone.